src/actions/userActions.js

```diff
@@ -88,10 +88,12 @@ export const getUserByEmail = () => dispatch => {
       dispatch({
         type: types.GET_USER_BY_EMAIL,
         payload: res.data
-      });      
-      if(res.data.role == 'admin' || res.data.role == 'vendor'){
-        dispatch(clearFilteredProducts())
-      }
+      });
+
+      // why?
```

Comment:
filtered products were (now are) persisting after login, Ie: if an admin had performed a search or filtered by category prior to logging in, that search would be reflected in the admin portal products list.